kiliautoml/models/_hugging_face_text_classification_model.py

```diff
@@ -35,7 +35,7 @@ class HuggingFaceTextClassificationModel(BaseModel, HuggingFaceMixin, KiliTextPr
 
     ml_task: MLTaskT = "CLASSIFICATION"
     model_repository: ModelRepositoryT = "huggingface"
-
+    model_framework: ModelFrameworkT = "pytorch"
```

Comment:
isn't it possible to use "tensorflow" with hugging face ?

Reply:
Yes, but it's not implemented for the moment